### ISSUE TITLE
Interaction bug between Chromaticraft pylons and Bound armor

### ISSUE THREAD

**Issue by Darktemplarx** [WayofTime/BloodMagic#445]
So, idk if you've played with Reika's Chromaticraft, but recently he added an effect when his pylons shoot at you that burns a little LP out of your soul network. Normally this isn't that big of a deal, but I find that when you're wearing a full suit of Bound armor, a single pylon surge one hit kills you and sets your bound armor to about -3000/1000 durability. Re equipping it has the usual effect of consuming LP to repair (it takes a LOT).

When I reported this bug to Reika, he said "The kind of drain given is the "non spilling to damage" kind, at least as per the API's documentation. Ask WayOfTime if that still works."

Tested with Forge 1490, Blood Magic 1.3.3-13, and Chromaticraft V8e, although this behavior was added back in V4 which was released Feb 13, 2015.

**Comment by Arcaratus** [comment]
What's the pylons name?

**Comment by Darktemplarx** [comment]
They're just called crystal pylon. They are part of worldgen, they look like this: http://i.imgur.com/O2fUnQH.jpg?1, and they come in all of minecraft's 16 colors.

**Comment by bookerthegeek** [comment]
If that is the intended mechanic, ouch. But nice I guess for pvp servers.

**Comment by Darktemplarx** [comment]
I'm pretty sure it's not intended. Also it would be very hard to weaponize in PVP, because pylons both cannot be created by players (they spawn as part of the world) and cannot be moved by any means.
Also looking back at OP, I guess February is not "recently" at all.

Back on topic, I would like to know how bound armor works. I know that if it takes durability damage, it uses some of your Soul Network to repair itself. But does it also burn your soul network to reduce incoming damage? Or does something burning some of your soul network do something weird to the armor, causing extra damage?

**Comment by bookerthegeek** [comment]
I'd just build my base around it. 

From past interaction, the bound armour is not supposed to take damage. (I know it does, it's their to prevent a crash when for an odd reason it gets damaged. Good example is to load up tppi  and get hit with an infiminer. It would damage the armor and the only way to fix it was to unbind the armour and then return bind it.)

The bound armour reduces incoming damage by applying it to your soul network.

**Comment by TheMadmanofChaos** [comment]
Booker, Bound armor always takes damage, from every hit you take. It just instantly refiles the missing durability, charging LP to do so.

To the main post: This is a rather odd issue. Unless the pylon is hardcoded to check for bound armor and deal more damage as a result, this shouldn't happen. The "Extra LP drain" from getting hit should just act like you had a sigil used, and that shouldn't damage the armor(Worse case scenario, you run out of LP and directly lose hearts to feed your armor.) Even in this case, the only way to lose THAT much durability is if you got hit with a lot of damage. As in a Tinkers "Bane of Pigs", or a small nuke. You would need to take thousands of points of damage for the armor to be hurt that badly.

**Comment by Darktemplarx** [comment]
I did some testing on my own, and I got some clarification from Reika regarding this matter.
Firstly, a pylon deals 2.5 hearts of damage in a normal surge. It deals 25 hearts of damage if you're wearing bound armor. Assuming the classic ratio of 1 heart to 200 LP applies to bound armor absorbing damage, this is where the extra LP burn comes from.
Second, I was told that pylons do specifically check for bound armor, and drain from the soul network, but Reika didn't tell me anything about the intended numbers.
I still can't understand why the armor takes so much durability damage.
